probe-rs/src/probe.rs

```diff
@@ -419,6 +420,15 @@ impl Probe {
         }
     }
 
+    /// Configure the scan chain to use.
```

Comment:
Can we add a refrence to the trait method here so people get the full picture or even maybe copy paste the docs and add the bit that this is only callable before attaching :)

Reply:
Definitely. Good ideas.